From e9f9ca60cdb16cb16937d55d24ffb3b5f0f6f1dd Mon Sep 17 00:00:00 2001
From: "Dmitry V. Levin" <ldv@altlinux.org>
Date: Mon, 3 Jan 2022 04:24:02 +0300
Subject: [PATCH 0752/2065] [Backport] usr/include/Makefile: add linux/nfc.h to
 the compile-test coverage

stable inclusion
from stable-v5.10.96
commit 8bae6db29c7f2d0452bf721d6c6b8cf28b6eee35
bugzilla: https://gitee.com/openeuler/kernel/issues/I55NWB

Reference: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=8bae6db29c7f2d0452bf721d6c6b8cf28b6eee35

--------------------------------

commit 10756dc5b02bff370ddd351d7744bc99ada659c2 upstream.

As linux/nfc.h userspace compilation was finally fixed by commits
79b69a83705e ("nfc: uapi: use kernel size_t to fix user-space builds")
and 7175f02c4e5f ("uapi: fix linux/nfc.h userspace compilation errors"),
there is no need to keep the compile-test exception for it in
usr/include/Makefile.

Signed-off-by: Dmitry V. Levin <ldv@altlinux.org>
Signed-off-by: Masahiro Yamada <masahiroy@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Yu Liao <liaoyu15@huawei.com>
---
 usr/include/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/usr/include/Makefile b/usr/include/Makefile
index f6b3c85d900e..703a255cddc6 100644
--- a/usr/include/Makefile
+++ b/usr/include/Makefile
@@ -34,7 +34,6 @@ no-header-test += linux/hdlc/ioctl.h
 no-header-test += linux/ivtv.h
 no-header-test += linux/kexec.h
 no-header-test += linux/matroxfb.h
-no-header-test += linux/nfc.h
 no-header-test += linux/omap3isp.h
 no-header-test += linux/omapfb.h
 no-header-test += linux/patchkey.h
-- 
2.17.1

